src/util/permissionIdentities.tsx

```diff
@@ -5,6 +5,11 @@ export type ChangeSummary = Record<
   { added: Set<string>; removed: Set<string>; name: string }
 >;
 
+export type FineGrainedTLSFormValues = {
+  name: string;
+  publicity: boolean;
```

Comment:
```suggestion
```
I think this is not a valid field and unused.

Reply:
I will remove the entire type as the rename functionality is not being introduced in this PR.